### PR TITLE
Add two more pages to catalan story

### PR DESCRIPTION
**Pull request by eduardbosch**
Please add this new adventure path for Catalan story :), @casey-collab

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2767)

### CODE REVIEW

File: catala/dormir/dormir.md [hunk 1]
```diff
@@ -1,2 +1,7 @@
 Et quedes dormint com un suro. A les 14 hores
-et desperta ta mare amb un plat d'arròs al forn.
\ No newline at end of file
+et desperta ta mare amb un plat d'arròs al forn.
+
+Veus que surt fum del plat. Què fas?
+
+[Tasto una mica](tastar/tastar_arros.md)
+[Pregunto a la mare per què surt fum del plat](preguntar/preguntar_fum.md)
```
Comment (casey-collab) [inline catala/dormir/dormir.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by eduardbosch** [comment]
Thanks @casey-collab!

**Comment by casey-collab** [comment]
Looks great! :rabbit:

**Pull request merged**